api/src/routes/inference/inference.ts

```diff
@@ -63,17 +61,22 @@ app.post(
       );
     }
 
+    const provider = inferenceConfig.aiProviderType;
+
     // Expand out of scope identifiers in the handler function, to add as additional context
     //
     // Uncomment console.time to see how long this takes
     // It should be slow on the first request, but fast-ish on subsequent requests
     //
     // console.time("Handler and Middleware Expansion");
     const [handlerContextPerformant, middlewareContextPerformant] =
-      await expandHandler(handler, middleware ?? []).catch((error) => {
-        logger.error(`Error expanding handler and middleware: ${error}`);
-        return [null, null];
-      });
+      // HACK - Ditch the expand handler for ollama for now, it overwhelms llama 3.1-8b
```

Comment:
for the purposes of a demo, use a simplistic app and turn off static analysis.

keeps response times lower, and prevents the model from getting confused about tool calls (from my limited testing)

Reply:
however, it was able to handle smaller bits of additional context about handler functions, fwiw